pkgs/development/python-modules/stravaweblib/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  arrow,
```

Comment:
Not used. I recommend using https://github.com/nix-community/nixd to detect such issues before committing.

Reply:
Fixed in f12a3771ac27

Thanks for the tip about nixd :) Maybe we should have a CI check testing such cases?

Reply:
> Fixed in [f12a377](https://github.com/NixOS/nixpkgs/commit/f12a3771ac270d1abb197936232061ddb0129301)
> 
> Thanks for the tip about nixd :) Maybe we should have a CI check testing such cases?

I think I once saw a discourse post saying that similarly to how checking whether files are formatted in CI, people want to get such diagnostics as well, but I can't find a link to such a post / comment.

Reply:
It was previously introduced in nixpkgs, but currrently disabled in [this issue](https://github.com/NixOS/nixpkgs/issues/332695) because some people want to use a CLI frontend.

Reply:
> It was previously introduced in nixpkgs, but currrently disabled in [this issue](https://github.com/NixOS/nixpkgs/issues/332695) because some people want to use a CLI frontend.

Thanks for the link!